tools/benchmark/cmd/put.go

```diff
@@ -134,21 +134,21 @@ func compactKV(clients []*v3.Client) {
 		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 		resp, err := c.KV.Get(ctx, "foo")
 		cancel()
-		if err != nil {
```

Comment:
The checker complains with `tools/benchmark/cmd/put.go:141:3: the surrounding loop is unconditionally terminated (SA4004)`

I will mask this error code. I think current implementation is fine too.